### PR TITLE
Offramp Pages + Yes/No Radio Buttons (Conf. Name Page)

### PR DESCRIPTION
**Pull request by nmakuch**
### 1) Creating the `offramp-name` page
| offramp-name page |
|--------|
|    offramp-name    |

#### Breakdown:
- Created page + routes
- Copied content from designs into new pages
- Updated `personal.controller` test that used the old offramp page to use "/offramp-name"

### 2) Switched from confirm buttons to yes/no radio buttons on `/personal/name` 
| Before |  After   |
|--------|--------|
|      Screen Shot 2019-07-26 at 15 07 52 . |     Screen Shot 2019-07-26 at 15 07 59       |

#### Breakdown:
- Switched from buttons to yes/no radio buttons
- 'Yes' will continue through the normal flow
- 'No' will bring the user to the new `offramp-name` page
- Removed help dropdown from `/personal/name` because of new offramp page

### 3) Creating the `offramp-residence` page
| offramp-residence page |
|--------|
|    offramp-residence    |

#### Breakdown:
- Created page + routes
- Copied content from designs into new pages
- Updated `personal.controller` test that used the old offramp page to use "/offramp-residence"

### 4) Creating the `offramp-financial` page
| offramp-financial page |
|--------|
|    offramp-financial    |

#### Breakdown:
- Created page + routes
- Copied content from designs into new pages
- Updated `personal.controller` test that used the old offramp page to use "/offramp-financial"

### CODE REVIEW

File: public/scss/components/_offramp.scss [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      p {
+          margin-bottom: $space-md;
+      }
+
+      ul li {
+          margin-bottom: $space-xxs;
+        }
+      
+    } 
+}
```
Comment (pcraig3) [inline public/scss/components/_offramp.scss]: In general, we should have general `p` tag rules that work across the whole app than just for these pages. 

I have a PR in the works that changes some of our app-wide text formatting. Let's take this out for now.

File: views/offramp/offramp-residence.pug [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  
+  .offramp
+    p #{__('Since you weren’t a resident of Ontario on December 31, 2018, you can’t use this service to file your taxes.')}
+
+    div
+      p #{__('There are also other free and secure ways to file your taxes:')}
+      ul
+        li <a href=#{__('https://www.canada.ca/en/revenue-agency/services/tax/individuals/community-volunteer-income-tax-program.html')}>#{__('Visit a free tax clinic near you. ')}</a> #{__('You can have a volunteer help you submit your taxes.')}
+        li #{__('Contact CRA at')} #[a(href="tel:123-456-7890") (123) 456-7890] #{__('for help')}
+        li #{__('File your taxes with')} <a href=#{__('https://www.canada.ca/en/revenue-agency/services/e-services/e-services-individuals/netfile-overview/certified-software-netfile-program.html')}>#{__('other NETFILE software certified by CRA')}</a>
```
Comment (pcraig3) [inline views/offramp/offramp-residence.pug]: These bullet points have been repeated several times. Can you put this into an includes maybe?

Also, our current offramp page has a bullet list that looks like this. Can you confirm with Brian whether we can reuse the bullet points we have on the other offramp page?

File: public/scss/components/_offramp.scss (hunk 1 above)
Comment (pcraig3) [reply public/scss/components/_offramp.scss]: Save it for later though and we can see if it's needed.

File: views/personal/name.pug [hunk 3]
```diff
@@ -21,12 +22,10 @@ block content
...
             div #{data.personal.firstName} #{data.personal.lastName}
+  
+  form.cra-form(method='post')
+    +yesNoRadios('name', null, 'Is this your name?', errors)
 
-  div
-    details.
-      <summary><span>#{__('Help with name confirmation')}</span></summary>
-      <p>#{__('If you have changed your name since the last time you filed your taxes,')} <a href=#{__('https://www.canada.ca/en/revenue-agency/services/tax/individuals/frequently-asked-questions-individuals/should-you-tell-cra-about-your-change-name.html')} target="_blank">#{__('get in contact with the CRA to update your information')}</a>.</p>
-      <p>#{__('Once updated, you can proceed to use this service.')}</p>
```
Comment (pcraig3) [inline views/personal/name.pug]: Should we link to any of the CRA guidance around name changes on the offramp page?

Comment (pcraig3) [reply views/personal/name.pug]: Also, on the `/financial/income` page, we have a revealing element that shows up if you're about to say "no". Do we need that here, @brianhendrick1?

File: views/offramp/offramp-name.pug [hunk 4]
```diff
@@ -0,0 +1,20 @@
+extends ../base
+include ../_includes/banner
+
+block variables
+  -var title = __('Continue with an incorrect name on file')
```
Comment (pcraig3) [inline views/offramp/offramp-name.pug]: This title suggests that you can continue using this service somehow. Is there no better way to word this? @brianhendrick1.

File: views/personal/name.pug (hunk 3 above)
Comment (nmakuch) [reply views/personal/name.pug]: Brian and I had talked about this a little bit, and I think we're going to leave it in for now until we think more on it.

File: views/offramp/offramp-residence.pug (hunk 2 above)
Comment (nmakuch) [reply views/offramp/offramp-residence.pug]: I think the first bullet on the other offramp page would become obsolete with the new offramp pages. If we could delete it, all bullets would be consistent, I'll double check with Brian

**Pull request merged**